templates/contact.html

```diff
@@ -0,0 +1,222 @@
+        <div class="grid grid-cols-1 gap-x-8 gap-y-6 sm:grid-cols-2">
+        <div class="sm:col-span-2">
+            <label for="company" class="block text-sm/6 font-semibold text-gray-900">Company</label>
+            <div class="mt-2.5">
+                <input type="text" name="company" id="company" class="block w-full rounded-md border-0 px-3.5 py-2 text-gray-dark shadow-sm ring-1 ring-inset ring-gray-dark placeholder:text-gray-dark focus:ring-2 focus:ring-inset focus:ring-gray-dark sm:text-sm/6">
+            </div>
+        </div>
+          <div>
+            <label for="first-name" class="block text-sm/6 font-semibold text-gray-dark">First name</label>
+            <div class="mt-2.5">
+              <input type="text" name="first-name" id="first-name" autocomplete="given-name" class="block w-full rounded-md border-0 px-3.5 py-2 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-dark placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-gray-dark sm:text-sm/6">
+            </div>
+          </div>
+          <div>
+            <label for="last-name" class="block text-sm/6 font-semibold text-gray-900">Last name</label>
+            <div class="mt-2.5">
+              <input type="text" name="last-name" id="last-name" autocomplete="family-name" class="block w-full rounded-md border-0 px-3.5 py-2 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-dark placeholder:text-gray-dark focus:ring-2 focus:ring-inset focus:ring-gray-dark sm:text-sm/6">
+            </div>
+          </div>
+          <div class="sm:col-span-2">
+            <label for="email" class="block text-sm/6 font-semibold text-gray-900">Email</label>
+            <div class="mt-2.5">
+              <input type="email" name="email" id="email" autocomplete="email" class="block w-full rounded-md border-0 px-3.5 py-2 text-gray-dark shadow-sm ring-1 ring-inset ring-gray-dark placeholder:text-gray-dark focus:ring-2 focus:ring-inset focus:ring-gray-dark sm:text-sm/6">
+            </div>
+          </div>
+          <div class="sm:col-span-2">
+            <label for="message" class="block text-sm/6 font-semibold text-gray-dark">Message</label>
+            <div class="mt-2.5">
+              <textarea name="message" id="message" rows="4" class="block w-full rounded-md border-0 px-3.5 py-2 text-gray-dark shadow-sm ring-1 ring-inset ring-gray-dark placeholder:text-gray-dark focus:ring-2 focus:ring-inset focus:ring-gray-dark sm:text-sm/6"></textarea>
+            </div>
+          </div>
+        </div>
+        <div class="mt-8 flex justify-end">
+          <button id="send-email" type="button" class="rounded-md bg-blue px-3.5 py-2.5 text-center text-sm font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue">Send message</button>
+        </div>
+      </div>
+      <div id="emailSuccess" class="mt-2 rounded-md bg-green-success-light p-4 hidden">
```

Comment:
```suggestion
      <div id="emailSuccess" class="mt-2 rounded-md bg-green-success-light p-4 mx-auto max-w-xl lg:mr-0 lg:max-w-lg hidden">
```
Just to align the confirmation message:

Before: 
<img width="729" alt="image" src="https://github.com/user-attachments/assets/4ff7d216-872e-4ca6-b4df-f202de251718">

After:
<img width="665" alt="image" src="https://github.com/user-attachments/assets/71c11402-1023-42a0-ad3b-69e6fa935c38">